src/mapfile/mapfile.l

```diff
@@ -538,13 +538,14 @@ static void end_layer()
 		if (!map_l->storage->schema->use)
 			buffer_add_str(map_l->storage->schema, "public");
 		if (!map_is_dump) map_l->retrievable=map_l->writable = false;
-
+                        map_l->name_prefix = buffer_init();
```

Comment:
Should probably use spaces for indentation instead of tabs, the alignment doesn't match.

Reply:
thanks, updated